modules/s3-remote-state/main.tf

```diff
@@ -36,12 +35,26 @@ variable "force_destroy" {
   type        = bool
 }
 
+variable "kms_key_id" {
+  description = "The ARN of a KMS Key to use for encrypting the state"
+  type        = string
+}
+
```

Comment:
The use of a KMS key here is a pattern worth encoding into a module like this. TY for this update.

I guess the complimentary update is a new module which defines the KMS key to be used by this module.

Reply:
That sounds good to me. I've opened https://github.com/fpco/terraform-aws-foundation/issues/310 and included that detail in the issue.